api/api/admin/media_report.py

```diff
@@ -135,131 +381,147 @@ def get_changelist(self, request, **kwargs):
 
 
 class MediaReportAdmin(admin.ModelAdmin):
-    change_form_template = "admin/api/media_report/change_form.html"
-    list_display = ("id", "reason", "is_pending", "description", "created_at", "url")
+    media_type = None
+
+    @admin.display(description="Is pending?", boolean=True)
+    def is_pending(self, obj):
+        """Shadow the ``is_pending`` property but render as icon in Django admin."""
```

Comment:
What does "shadow"ing the property mean?

Reply:
If this function didn't exist, `is_pending` would still be rendered in the Django admin (because it is a `@property` on the Django models for reports). By defining this function, we told Django Admin to use this instead of the property, and show checks/crosses instead of "True"/"False" that would appear if we hadn't defined it here.

Reply:
Is shadow a Django term of art? Or just meant to indicate that the function is defined to tell Django admin about the property on the model?

Reply:
> I understand the purpose of the function

Of course 🤦! Sorry I understood otherwise. As for shadowing, I was just using the PyCharm IDE terminology here.

<img width="940" alt="Screenshot 2024-05-28 at 11 05 12 AM" src="https://github.com/WordPress/openverse/assets/16580576/68fbb3bf-b834-4a62-8d81-2cd5295ade8d">

I can change the term to a more generic one if shadowing feels incorrect or has some negative subtext.

Reply:
Ohhhh I see! Pycharm is referring to "variable shadowing", which is a specific and technical term: https://en.wikipedia.org/wiki/Variable_shadowing. In that case, the `id` variable shadows the globally available `id` function in Python: https://docs.python.org/3/library/functions.html#id

That isn't happening with `is_pending` here, there's no "shadowing" possible with a class-function declaration in Python (though it is possible with class attributes).

The verb shadow here is confusing to me I guess because of these different meanings of it, and how it can't apply to a function in Python. It's not quite an override, but I think you could just change the comment to "Set an explicit display type for the property, otherwise Django does not know the type of a model property".

Reply:
Thanks for the explanation, I've updated the text like you suggested in 50c420c.